colophon.php

```diff
@@ -62,14 +62,14 @@ function team51_credits( $args = array() ) {
 	}
 
 	if ( $args['pressable'] ) {
-		$pressable_link            = apply_filters(
+		$pressable_link = apply_filters(
 			'team51_credits_link_pressable',
 			add_query_arg(
 				array(
 					'utm_source'   => 'Automattic',
 					'utm_medium'   => 'rpc',
```

Comment:
Just out of curiosity: 
- Why the `utm_medium` is different for the 'pressable' case? Isn't the link rendered as well from the colophon output?
- Why aren't we including the `partner_domain` parameter in this case? Is it only needed for wpcom?
- Would it make sense to generate and return this array from a different function to avoid duplicate code? Especially if we need to add more use cases in the future.

Reply:
It's different because this link was simply duplicating the link that Pressable had given us to put on footer backlinks initially.

The colophon didn't add extra data to them intentionally, just mirrored the existing markup.  I'm fine adding more data if Pressable folks would make use of it -- but adding the partner domain in here feels somewhat superfluous due to them already knowing where it came from via the http referrer, no?

Reply:
Yeah, I think we can ignore the `utm_medium` attribute.